Open Judge System/LocalWorker/OJS.LocalWorker/OjsSubmissionProcessingStrategy.cs

```diff
@@ -220,11 +230,6 @@ private void SaveParticipantScore()
 
                 var participant = this.participantsData
                     .GetByIdQuery(this.submission.ParticipantId.Value)
-                    .Select(p => new
-                    {
-                        p.IsOfficial,
-                        p.User.UserName
-                    })
                     .FirstOrDefault();
```

Comment:
Include all the necessary data with single request to avoid lazy loading
```
var participant = this.participantsData
    .GetByIdQuery(this.submission.ParticipantId.Value)
    .Select(p => new
    {
        Participant = p,
        UserName = p.User.UserName,
        TotalScore = p.Scores
            .Where(ps => !ps.Problem.IsDeleted)
            .Select(ps => ps.Points)
            .Sum()
    })
    .FirstOrDefault();
```

Reply:
Done